trunk/SUAVE/Components/Component.py

```diff
@@ -22,7 +22,7 @@ class Component(Data):
     def __defaults__(self):
         self.tag    = 'Component'
         self.origin = [[0.0,0.0,0.0]]
-    
+        #self.origin = np.array([0,0,0])

```

Comment:
I think it should be removed if it's not functional. If it is or you want to keep it for other reasons I think an explanation should be added.